railties/lib/rails/commands/server/server_command.rb

```diff
@@ -72,18 +72,6 @@ def create_tmp_directories
         end
       end
 
-      def log_to_stdout
-        wrapped_app # touch the app so the logger is set up
```

Comment:
Hmm, not sure were this method is defined and why it's no longer necessary. Any idea 😄

Reply:
Good question!

`wrapped_app` comes from `Rack::Server` here: https://github.com/rack/rack/blob/abca7d59c566320f1b60d1f5224beac9d201fa3b/lib/rack/server.rb#L421

Rack calls `wrapped_app` on its own here: https://github.com/rack/rack/blob/abca7d59c566320f1b60d1f5224beac9d201fa3b/lib/rack/server.rb#L327

However, if you want to modify the application (like making the logger broadcast), then you have to "touch the app". All this does is load the app a bit earlier than Rack would normally so that things like initializers have a change to run first.